### DISCUSSION TITLE
New Superweapon

### DISCUSSION THREAD

**Discussion by Daiki9527** [Phobos-developers/Phobos#1028] [Ideas and feedback]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

When the superweapon is launched, specified units attack the locaion where the superweapon is fire at.

### INI code

```ini
[SomeSuperWeapon]
Type=BeaconAttack
BeaconAttack.Types=TechnoTypes1,TechnoTypes2
BeaconAttack.Counts=1,2;1 TechnoTypes1 and 2 TechnoTypes2 Fire at the location.
BeaconAttack.PickRandom=yes,yes
BeaconAttack.FireWeapon=0;0 For Primary 1 For Secondary.
BeaconAttack.CheckRange=yes/no
```

**Comment by xindun110** [comment]
Very good creativity, very good!
